File: FirstBitSolutions/C Module/Assignments/Assignment 6/Func_Assignment_3/FunctionsType3/Ques8.c
//8 Check the given number is Strong number or not.
//Input: n = 145
//Output: Strong
#include<stdio.h>
void Strong(int);//Declaration
void main()
{
	int n=188;
	Strong(n);//Call
}//main ends here
void Strong(int a){

	int i =a,fact=1,temp,sum;
	while(i>a){
		fact = fact * i;
		temp = fact;
		i--;
		
	}
	sum = fact + temp;
	if(sum == a)
		printf("It is strong");
	else
		printf("It is not strong");
}//strong ends here

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 3/Ques8.java
class Ques{ 
public static void main(String args[]){ 

	int n=188;
	int i =n,fact=1;
	int temp=0,sum;
	while(i>n){
		fact = fact * i;
		temp = fact;
		i--;
		
	}
	sum = fact + temp;
	if(sum == n)
		System.out.printf("It is strong");
	else
		System.out.printf("It is not strong");
 }
}